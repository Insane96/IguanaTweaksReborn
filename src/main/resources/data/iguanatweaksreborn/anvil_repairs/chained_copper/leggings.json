{
	"item_to_repair": {
		"id": "iguanatweaksreborn:chained_copper_leggings"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 5
		}
	]
}