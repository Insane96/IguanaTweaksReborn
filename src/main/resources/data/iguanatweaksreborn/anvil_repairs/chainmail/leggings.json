{
	"item_to_repair": {
		"id": "minecraft:chainmail_leggings"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:chain"
			},
			"amount": 5
		}
	]
}