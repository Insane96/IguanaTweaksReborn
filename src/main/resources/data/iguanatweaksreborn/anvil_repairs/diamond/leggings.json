{
	"item_to_repair": {
		"id": "minecraft:diamond_leggings"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 6
		}
	]
}