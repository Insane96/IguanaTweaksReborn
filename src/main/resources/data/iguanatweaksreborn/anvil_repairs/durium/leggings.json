{
	"item_to_repair": {
		"id": "iguanatweaksreborn:durium_leggings"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 7,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:durium_ingot"
			},
			"amount": 2
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:durium_nugget"
			},
			"amount": 18
		}
	]
}