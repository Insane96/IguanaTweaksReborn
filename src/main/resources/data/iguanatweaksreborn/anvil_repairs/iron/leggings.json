{
	"item_to_repair": {
		"id": "minecraft:iron_leggings"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 6
		},
		{
			"repair_material": {
				"id": "minecraft:iron_nugget"
			},
			"amount": 50,
			"cost_multiplier": 9
		}
	]
}