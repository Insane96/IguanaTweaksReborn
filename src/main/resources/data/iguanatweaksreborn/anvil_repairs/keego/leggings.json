{
	"item_to_repair": {
		"id": "iguanatweaksreborn:keego_leggings"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 7,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:keego"
			},
			"amount": 2
		}
	]
}