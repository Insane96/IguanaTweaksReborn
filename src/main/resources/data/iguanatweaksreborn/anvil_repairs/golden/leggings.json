{
	"item_to_repair": {
		"id": "minecraft:golden_leggings"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:gold_ingot"
			},
			"amount": 6
		},
		{
			"repair_material": {
				"id": "minecraft:gold_nugget"
			},
			"amount": 50,
			"cost_multiplier": 9
		}
	]
}